crates/primitives/src/l1/btc.rs

```diff
@@ -740,6 +741,92 @@ impl XOnlyPk {
     }
 }
 
+impl From<Transaction> for RawBitcoinTx {
+    fn from(value: Transaction) -> Self {
+        Self(serialize(&value))
+    }
+}
+
+impl TryFrom<RawBitcoinTx> for Transaction {
+    type Error = encode::Error;
+    fn try_from(value: RawBitcoinTx) -> Result<Self, Self::Error> {
+        deserialize(&value.0)
+    }
+}
+
+impl TryFrom<&RawBitcoinTx> for Transaction {
+    type Error = encode::Error;
+    fn try_from(value: &RawBitcoinTx) -> Result<Self, Self::Error> {
+        deserialize(&value.0)
+    }
+}
```

Comment:
Can you add trivial tests:

- case when the transaction is valid in bytes and the `TryFrom` succeeds
- case when the transaction is invalid in bytes and the `TryFrom` fails